test/rematch.jl

```diff
@@ -1,5 +1,4 @@
-using Compat.Test
-using Compat.InteractiveUtils
+using Test
```

Comment:
Should we remove the dependency on `Compat` and just require Julia 1.0?

Reply:
👍 yeah, agreed it'd be good to do that too. It's used in the `src/` still, so yeah we'd have to bump to a later julia version.

I'll leave that for a later change though, so i can get this merged 😊